spec.bs

```diff
@@ -210,6 +212,8 @@ The <dfn attribute for=Subscriber><code>signal</code></dfn> getter steps are to
 <div algorithm>
   The <dfn for=Subscriber method><code>error(|error|)</code></dfn> method steps are:
 
+    1. If [=this=]'s [=Subscriber/active=] is false, then return.
```

Comment:
This is correct, but there's an additional step. If the `Subscriber/active` is `false`, we'll want to report the `|error|` with `reportError`, because it can't be handled and we don't want it to be completely swallowed or ignored. cc @domfarolino

Reply:
Right, that's what we do right now: https://wicg.github.io/observable/#dom-subscriber-error:~:text=Otherwise%20(i.e.%2C%20when%20error%20algorithm%20is%20null)%2C%20report%20the%20exception%20error

Reply:
It's currently complicated in this PR because https://github.com/WICG/observable/pull/129/#issuecomment-2033200801 has not been addressed yet. If we address https://github.com/WICG/observable/pull/129/#issuecomment-2033200801, then we can use the `active` variable as the sole signal for whether we should proceed or not, instead of sometimes use `active` and sometime use "algorithm nullness".